keystone/files/keystonerc

```diff
@@ -7,3 +7,4 @@ export OS_REGION_NAME={{ server.region }}
 export OS_SERVICE_TOKEN={{ server.service_token }}
 export OS_SERVICE_ENDPOINT="http://{{ server.bind.private_address }}:{{ server.bind.private_port }}/v2.0/"
 export OS_ENDPOINT_TYPE="internal"
+export OS_CACERT="/etc/ssl/certs/ca-certificates.crt"
```

Comment:
This will work only on Debian-based distributions. You should use map.jinja and point RHEL to /etc/pki/tls/certs/ca-bundle.crt